server/util/redact/redact.go

```diff
@@ -47,7 +47,7 @@ const (
 )
 
 var (
-	envVarOptionNames      = []string{"client_env", "repo_env", "test_env"}
+	envVarOptionNames      = []string{"action_env", "client_env", "repo_env", "test_env"}
```

Comment:
Could you also add [`--host_action_env`](https://bazel.build/reference/command-line-reference#flag--host_action_env)?

Reply:
Oh yeah, thanks for the tip. Done!